src/deployment-guides/tko-aws-airgap.md

```diff
@@ -0,0 +1,375 @@
+
+ **Step 01. Create a VPC**
+
+* Login to your AWS Console and select vpc service.
+* Create your VPC with Valid CIDR and name.
+
+**Step 02.  Create 3 Private Subnets**
+
+* Click Subnet and create your Subnet with:
+* Private Subnet 1 , Private Subnet 2 and Private Subnet 3 valid Name & VPC.
+* Valid Subnet range which is valid IPv4 CIDR Block.
+
+**Step 03. Create Private Route Table**
+
+* Create a Route table in the same VPC.
+* Make sure you selected the right VPC and give a proper tag.
+
+**Step 04. Add Private Subnet in Private Route Table**
+
+* Edit the Subnet Association.
+* Select the PrivateSubnet checkbox.
+* Click on the Save button.
+
+**Step 05. Edit DNS Resolution and Hostname**
+* Click on Action and Edit DNS hostname
+* Checkmark on DNS Hostname and click on save.
+
+### Add VPC endpoints into offline VPC
+
+After you create the offline VPC, you must add following endpoints to it (VPC endpoint enables private connections between your VPC and supported AWS services):
+
+* Service endpoints:
+* sts
+* ssm
+* ec2
+* ec2messages
+* elasticloadbalancing
+* secretsmanager
+* ssmmessages
+* s3(gateway type)
+
+You can refer [aws create endpoints service documentation](https://docs.aws.amazon.com/vpc/latest/privatelink/create-endpoint-service.html) for further details.
+
+## <a id=offline-jumpbox> </a> Creating offline JumpBox
+
+After doing the network configuration, complete the steps described in this section to set up your jumpbox. You will download the Tanzu CLI to the jumpbox, which you will use to deploy the management cluster and workload clusters from the jumpbox. You also keep the Tanzu and Kubernetes configuration files for your deployments on your jumpbox.
+
+1. Create a jumpbox.
+
+	<!-- /* cSpell:disable */ -->
+
+	```bash
+   export vpcId=<offline vpc id>
+   export WORKING_DIR=<local working dir>
+
+	aws ec2 create-security-group --group-name "jumpbox-ssh" --description "To Jumpbox" --vpc-id "$vpcId" --output json > $WORKING_DIR/sg_jumpbox_ssh
+	aws ec2 create-tags --resources $(jq -r .GroupId $WORKING_DIR/sg_jumpbox_ssh) --tags Key=Name,Value="jumpbox-ssh"
+	# Allow ssh to jumpbox
+	aws ec2 authorize-security-group-ingress --group-id  $(jq -r .GroupId $WORKING_DIR/sg_jumpbox_ssh) --protocol tcp --port 22 --cidr "0.0.0.0/0"
+
+	# Save this file or use some team keypair already created
+	aws ec2 create-key-pair --key-name tkg-kp --query 'KeyMaterial' --output text > tkgkp.pem
+	chmod 400 tkgkp.pem
+
+	# Find an AMI for your region https://cloud-images.ubuntu.com/locator/ec2/ (20.04)
+	aws ec2 run-instances --image-id ami-036d46416a34a611c --count 1 --instance-type t2.xlarge --key-name tkg-kp --security-group-ids  $(jq -r .GroupId $WORKING_DIR/sg_jumpbox_ssh)   --subnet-id $(jq -r .Subnet.SubnetId $WORKING_DIR/subnet-pub-a)  --tag-specifications 'ResourceType=instance,Tags=[{Key=Name,Value=tkg-jumpbox}]' --block-device-mappings 'DeviceName=/dev/sda1,Ebs={VolumeSize=64}' > $WORKING_DIR/instance_jb_starting
```

Comment:
@sendjainabhi  I think we should refer to one of the private subnets which have been created above manually instead of reading via a file. 
$WORKING_DIR/subnet-pub-a
. What say?
cc @sunshetty


cc

Reply:
I think the way we specify it is less error prone then telling them to read it manually

Reply:
@z4ce - there is no public subnet in airgap ra , and we need to create jumpbox within private subnet. i will have to change this in instructions.

Reply:
i can see its env variable now, I just thought, as the user is creating the subnets manually. So he might not have subnet out file in working dir. But env variable works, I guess